src/xdocs/checks/design/hideutilityclassconstructor.xml

```diff
@@ -42,6 +42,27 @@ public class StringUtils // not final to allow subclassing
         </source>
       </subsection>
 
+      <subsection name="Properties" id="Properties">
+        <div class="wrapper">
+          <table>
+            <tr>
+              <th>name</th>
+              <th>description</th>
+              <th>type</th>
+              <th>default value</th>
+              <th>since</th>
+            </tr>
+            <tr>
+              <td>ignoreAnnotatedBy</td>
+              <td>Ignore classes annotated with the specified annotation(s). Annotation names provided in this property must exactly match the annotation names on the classes. If the target class has annotations specified with their fully qualified names (including package), the annotations in this property should also be specified with their fully qualified names. Similarly, if the target class has annotations specified with their simple names, this property should contain the annotations with the same simple names.</td>
+              <td><a href="../../property_types.html#String.5B.5D">String[]</a></td>
+              <td><code>{}</code></td>
+              <td>10.17.0</td>
```

Comment:
@romani @nrmancuso I think we have an issue on property versions. Another PR I looked at had 2 digit version, this one has 3. I looked at https://checkstyle.org/checks/annotation/annotationlocation.html#Properties and it has 2 digits.

We are missing some validation here and we need to correct it.

@Lmh-java Please create a new issue for this. We need to fix existing cases as well as add proper validation that everything is 3 digits if there isn't, imo.

Reply:
We use to have 2 digits release numbers. Only in 2023 we started doing 3 digits. Suppression list will be pretty long.

Reply:
I am resolving this.